### PR TITLE
Some rendering mappings

### PR DESCRIPTION
**Pull request by BleachDrinker420**
Mostly new mappings in `net/minecraft/client/renderer`

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -155,8 +155,6 @@ CLASS net/minecraft/class_197 net/minecraft/block/Block
 	FIELD field_478 boundingBoxMaxZ D
 		COMMENT Stores the Maximum value on the Z axis for the voxel shape
 	FIELD field_479 sound Lnet/minecraft/class_197$class_200;
-		COMMENT Stores the current block's sound type
-	FIELD field_479 soundGroup Lnet/minecraft/class_200;
```
Comment (arthurbambou) [inline mappings/net/minecraft/block/Block.mapping:159]: Why deleting it?

File: mappings/net/minecraft/client/render/GameRenderer.mapping [hunk 2]
```diff
@@ -1,67 +1,69 @@
...
+	FIELD field_10698 rainOffsetX [F
+	FIELD field_10699 rainOffsetY [F
 	FIELD field_10700 panoramaDirection I
 	FIELD field_10701 renderingPanorama Z
+	FIELD field_10702 shadersEnabled Z
+	FIELD field_10703 frameCount I
+	FIELD field_1813 thirdPersonDistance F
+	FIELD field_1814 lastThirdPersonDistance F
+	FIELD field_1819 cursorDeltaX F
+		COMMENT The {@code cursorDeltaX} amd {@code cursorDeltaY} fields store the actual mouse movement without smoothing, and are only used when smooth camera is enabled.
```
Comment (arthurbambou) [inline mappings/net/minecraft/client/render/GameRenderer.mapping:15]:
```suggestion
		COMMENT The {@code cursorDeltaX} and {@code cursorDeltaY} fields store the actual mouse movement without smoothing, and are only used when smooth camera is enabled.
```

File: mappings/net/minecraft/client/render/entity/EndermanEntityRenderer.mapping [hunk 3]
```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/class_549 net/minecraft/client/render/entity/EndermanEntityRenderer
-	FIELD field_6480 ENDERMAN Lnet/minecraft/class_1653;
 	FIELD field_2092 endermanModel Lnet/minecraft/class_438;
 	FIELD field_2093 random Ljava/util/Random;
+	FIELD field_6480 TEXTRE Lnet/minecraft/class_1653;
```
Comment (arthurbambou) [inline mappings/net/minecraft/client/render/entity/EndermanEntityRenderer.mapping:4]:
```suggestion
	FIELD field_6480 TEXTURE Lnet/minecraft/class_1653;
```

**Review by arthurbambou — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (arthurbambou) [reply mappings/net/minecraft/block/Block.mapping:159]: Duplicated fields aren't a problem anymore, in this case they don't have the same type descriptor that's why there are 2 entry for it.

Comment (BleachDrinker420) [reply mappings/net/minecraft/block/Block.mapping:159]: It wasn't building when i ran gradlew build so i thought it wasn't supposed to be there, my bad

**Review by arthurbambou — APPROVED** [review]

**Review by Grayray75 — APPROVED** [review]

**Comment by arthurbambou** [comment]
2 approvals + all check passed = merge

**Pull request merged**